Presentation.Web/app/components/global-admin/global-admin-local-admins-root.component.ts

```diff
@@ -0,0 +1,158 @@
+            },
+            controller: GlobalAdminLocalAdminRootController,
+            controllerAs: "ctrl",
+            templateUrl: `app/components/global-admin/global-admin-local-admins-root.view.html`
+        };
+    }
+
+    export interface ILocalAdminRow {
+        id: number,
+        organization: string,
+        name: string,
+        email: string,
+        objectContext: Models.Api.Organization.IAdminRightsDto,
+    }
+    
+    interface IGlobalAdminLocalAdminRootController extends ng.IComponentController {
+        currentOrganizationId: number;
+        userId: number;
+    }
+
+    class GlobalAdminLocalAdminRootController implements IGlobalAdminLocalAdminRootController {
+        currentOrganizationId: number | null = null;
+        userId: number | null = null;
+
+        localAdmins: Array<ILocalAdminRow> = [];
+        callbacks: IGlobalAdminLocalAdminCallbacks;
+
+        newUser: Models.ViewModel.Select2.IBaseSelect2OptionViewModel;
+        newOrg: Models.ViewModel.Select2.ISelect2OrganizationOptionWithCvrViewModel;
+
+        static $inject: string[] = ["userService", "organizationRightService", "select2LoadingService", "$scope", "notify"];
+        constructor(
+            private readonly userService: Services.IUserService,
+            private readonly organizationRightService: Services.Organization.IOrganizationRightService,
+            private readonly select2LoadingService: Services.ISelect2LoadingService,
+            private readonly $scope: ng.IScope,
+            private readonly notify) {
+        }
+        
+        $onInit() {
+            if (this.currentOrganizationId === null) {
+                throw "Missing parameter 'currentOrganizationId'";
+            }
+            if (this.userId === null) {
+                throw "Missing parameter 'userId'";
+            }
+
+            this.callbacks = {
+                removeRight: (rightId: number) => this.deleteLocalAdmin(rightId)
+            } as IGlobalAdminLocalAdminCallbacks;
+
+            this.loadData();
+        }
+
+        newLocalAdmin() {
+            // select2 changes the value twice, first with invalid values
+            // so ignore invalid values
+            if (typeof this.newUser !== "object") return;
+            if (!(this.newOrg && this.newUser)) return;
+
+            const userToUpdateId = this.newUser.id;
+            const newOrgId = this.newOrg.id;
+            var userName = this.newUser.text;
+            var orgName = this.newOrg.text;
+
+            const roleId = API.Models.OrganizationRole.LocalAdmin;
+
+            if (!(userToUpdateId && newOrgId && roleId)) return;
+            
+            var msg = this.notify.addInfoMessage("Arbejder ...", false);
+            const self = this;
+            this.organizationRightService.create(newOrgId, userToUpdateId, roleId)
+                .then(() => {
+                    msg.toSuccessMessage(userName + " er blevet lokal administrator for " + orgName);
```

Comment:
now that you added messages in the service, you can remmove the use of "notify" in this class. Otherwise the user gets multiple messages